File: Examples/Features/Formulas/ConfigWorkbookReferenceStyle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Formulas
{
    public class ConfigWorkbookReferenceStyle : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            //set workbook's reference style to R1C1. exported xlsx file will be R1C1 style.
            workbook.ReferenceStyle = ReferenceStyle.R1C1;
        }
    }
}

File: Examples.VB/Features/Formulas/ConfigWorkbookReferenceStyle.vb
Namespace Features.Formulas
    Public Class ConfigWorkbookReferenceStyle
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            'set workbook's reference style to R1C1. exported xlsx file will be R1C1 style.
            workbook.ReferenceStyle = ReferenceStyle.R1C1
        End Sub
    End Class
End Namespace
